Add support for CentOS 10 based distributions using Temurin JDK 17

mfuhrmann opened the pull request:
fix #55

mfuhrmann committed:
Update bootstrap-yum.sh

indigo423 commented on bootstrap-yum.sh:
```diff
@@ -246,15 +252,15 @@ installPostgres() {
 installOnmsApp() {
   echo -n "📦 Install OpenNMS Horizon packages      ... "
-  sudo dnf -y install rrdtool jrrd2 jicmp jicmp6 opennms-core opennms-webapp-jetty opennms-webapp-hawtio 1>>"${ERROR_LOG}" 2>>${ERROR_LOG}
+  sudo dnf -y install opennms  1>>"${ERROR_LOG}" 2>>${ERROR_LOG}
```
Can we keep the individual packages? The opennms meta package is a bit tricky to deal with. The resolution of 3rd party dependencies is sometimes a bit problematic.

mfuhrmann replied:
The original command failed. I didn't dive deeper into it and just followed the installation docs. We just install the `opennms` package there.
But I can give it a try again and check what exactly failed.

indigo423 commented on bootstrap-yum.sh:
```diff
@@ -223,19 +223,25 @@ EOF
+  gpgcheck=1
+  gpgkey=https://packages.adoptium.net/artifactory/api/gpg/key/public
+  EOF
   echo -n "📦 Install OpenJDK Development Kit       ... "
```
```suggestion
  echo -n "📦 Install Temurin Java Development Kit  ... "
```

indigo423 reviewed (commented):
Cool, just two very minor things.

mfuhrmann committed:
Update bootstrap-yum.sh

Co-authored-by: Ronny Trommer <ronny@no42.org>

indigo423 committed:
fix: Adoptium repo and opennms packages

Remove leading spaces from the Adoptium repository file and EOF
handling. Bring back the installation for individual opennms packages
instead the meta package which tries to resolve 3rd party dependencies.

Signed-off-by: Ronny Trommer <ronny@no42.org>

indigo423 reviewed (approved):
Fixed the CentOS/Rocky/RHEL support
